Terminal.Gui/View/Layout/DimAuto.cs

```diff
@@ -153,6 +153,11 @@ internal override int Calculate (int location, int superviewContentSize, View us
                                                            .ToList ();
                 }
 
+                foreach (View notDependentSubView in notDependentSubViews)
+                {
+                    notDependentSubView.SetRelativeLayout (us.Frame.Size);
```

Comment:
This should be `us.ContentView.Size`, right?

Also, did you try putting this inside the for loop below?

Reply:
> This should be `ContentSize`, right?
> 
I think you are right. I'll change that.

> Also, did you try putting this inside the for loop below?

Yes I did but doesn't work because it's run the layout and also calculating at same time for each subView and we need they run all the layout first for all subViews. Then the subViews already have the correct position and size to calculate the correct position and dimension for the superView.